Dockerfile

```diff
@@ -58,4 +60,44 @@ RUN cp ./target/sparkml-serving-3.3.jar /usr/local/lib/sparkml-serving-3.3.jar
 RUN cp ./serve.sh /usr/local/bin/serve.sh
 
 RUN chmod a+x /usr/local/bin/serve.sh
```

Comment:
Can all 3 of these `RUN` commands be combined into the same command?

Reply:
Yes, combining them.